4.analyze_data/notebooks/UMAP/image_settings/failing_drug_x_left.yaml

```diff
@@ -0,0 +1,56 @@
+failing-drug_x-left:
```

Comment:
Consider adding comments to this file or a README explaining the format and what each key represents??

Reply:
I assume the UMAP0 and UMAP1 are the coordinates and the paths are explanatory too